dev-tools/validation/test_implementation_capability_tracking.py

```
@@ -0,0 +1,271 @@
+                label,
+                lambda target=target: validator.linked_tracker_names(f"[tracker]({target})"),
+                "canonical relative target",
+            )
+        for label, target in (
+            ("unrelated dev-tools helper", "../../../dev-tools/validation/unrelated-helper.py"),
+            ("dev-tools README", "../../../dev-tools/README.md"),
+        ):
+            expect_failure(
+                label,
+                lambda target=target: validator.validate_evidence_anchor(
+                    root, owner, target, "proof", "proven", "[proof](x)", label
```

Ruff (0.15.21)

[warning] Function definition does not bind loop variable `label`

(B023)